README.md

```diff
@@ -4,37 +4,61 @@
 
 This repository contains the assets required to build the [Kubernetes website and documentation](https://kubernetes.io/). We're glad that you want to contribute!
 
-## Running the website locally using Hugo
+# Using this repository
 
-See the [official Hugo documentation](https://gohugo.io/getting-started/installing/) for Hugo installation instructions. Make sure to install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
+You can run the website locally using Hugo, or you can run it in a container runtime. We strongly recommend using the container runtime, as it gives deployment consistency with the live website.
 
-Before building the site, clone the Kubernetes website repository:
+## Prerequisites
 
-```bash
+To use this repository, you need the following installed locally:
+
+- [yarn](https://yarnpkg.com/)
+- [npm](https://www.npmjs.com/)
+- [Go](https://golang.org/)
```

Comment:
Why is `Go` included here? I'd be in favor of _removing_ this requirement.
For running the example tests and the linkchecker? You could add python to this
list if that is the reason to include Go. I don't think most contributors (first time) will run these apps.
Also, you need a specific version of Go to interoperate with Kubernetes (see the contribute docs).

Reply:
:+1: to removing Go here (and, separately, maybe add a page or README section about extra tools that can help with local testing).